riscv/src/lib.rs

```diff
@@ -152,7 +152,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-runtime = {{ path = "./runtime" }}
+powdr_riscv_rt = {{ git = "https://github.com/powdr-labs/powdr", branch = "main" }}
```

Comment:
Doesn't this also need the `runtime` subdirectory?

Reply:
What for?